tests/unit/engine.rs

```diff
@@ -357,7 +360,8 @@ mod tests {
             );
         }
         {
-            let engine = Engine::from_rules_debug(["||example.com^$script"], Default::default());
+            let engine =
+                Engine::from_rules_debug(["||example.com^$script"], Default::default());
```

Comment:
mind reverting the unrelated format changes? I will have to rebase this on our other branch as well

Reply:
won't it fail the format check then? That's why I ran it in the first place